### ISSUE TITLE
Filepath length and registry settings

### ISSUE THREAD

**Issue by rkaufman058** [PITT-HATLAB/plottr_clone#1]
This version of plottr will tend to make VERY long filepaths. It turns out that Windows will only allow 260 character filenames, and so the user will run into the issue of very long filepaths not being readable from their python scripts. This will show as a raised ValueError exception - "ValueError: Specified file does not exist."

Now _most_ of the time, this is because of a typo in your filename, forgotten backslash, etc. However if the filename is very long and you have triple-checked it, this might be the issue.

The long-term fix would be to avoid using longer filenames, and develop a mechanism for saving metadata in the plottr file itself rather than using the filename. 

The short-term bandaid solution is to change a windows registry setting that allows the use of longer filenames. That is documented [here](https://www.howtogeek.com/266621/how-to-make-windows-10-accept-file-paths-over-260-characters/). 

The short version, in case that page goes down, is to edit the following key:

Computer\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem\LongPathsEnabled 

to have the value 1.
